### PR TITLE
fix: address components copy icon on small screens

### PR DESCRIPTION
**Pull request by MukulKolpe**
## Description

Fixed the address components copy icon and `"connected address" + Address component` alignment for smaller screen size.

https://github.com/scaffold-eth/scaffold-eth-2/assets/78664749/51cbaf85-fe13-4500-8182-1d6503944d9d

## Additional Information

- [X] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [X] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #863_

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address: 0xfC001B20Db6195b148cfc4F7685091931D93cD93

### CODE REVIEW

**Comment by MukulKolpe** [comment]
@technophile-04, let me know if there are any changes that should be made.

File: packages/nextjs/components/scaffold-eth/Address.tsx [hunk 1]
```diff
@@ -92,50 +92,56 @@ export const Address = ({ address, disableAddressLink, format, size = "base" }:
...
   return (
     <div className="flex items-center">
-      <div className="flex-shrink-0">
-        <BlockieAvatar
-          address={checkSumAddress}
-          ensImage={ensAvatar}
-          size={(blockieSizeMap[size] * 24) / blockieSizeMap["base"]}
-        />
+      <div className="flex flex-col xs:flex-row items-center">
+        <div className="flex flex-row">
```
Comment (carletex) [inline packages/nextjs/components/scaffold-eth/Address.tsx:96]: I think we don't want the copy icon to go do a line wrap. We just want to not shrink. Probably you just need to add `flex-shrink-0` to the parent div.

In the particular case of the homepage (in your video, "connected address" + Address component) we'd just need to do flex-col on small resolutions and flex-row for the rest.

File: packages/nextjs/tailwind.config.js [hunk 2]
```diff
@@ -82,6 +82,9 @@ module.exports = {
       animation: {
         "pulse-fast": "pulse 1s cubic-bezier(0.4, 0, 0.6, 1) infinite",
       },
+      screens: {
+        xs: "400px",
+      },
```
Comment (carletex) [inline packages/nextjs/tailwind.config.js:87]: We won't need this.

**Comment by MukulKolpe** [comment]
Thank you, @carletex. I've made the suggested changes and updated the video. Please let me know if there are any further changes.

**Comment by technophile-04** [comment]
Hey thanks @MukulKolpe ! Added `flex-shrink-0` to both Icons as well since they were shrinking in smaller device in Transaction receipt : 

| Before | After | 
| ------- | ----- | 
| Screenshot 2024-06-17 at 11 51 13 AM |  Screenshot 2024-06-17 at 11 51 48 AM |

**Review by technophile-04 — APPROVED** [review]

**Pull request merged**